docs/content/en/docs/Customizing/configure_grub.md

```diff
@@ -130,12 +130,18 @@ Or to set the default entry to `fallback` system:
 
 By default `cOS` and derivatives shows the default boot menu entry while booting (`cOS`).
 
-The grub menu entry is generated during installation and can be configured by setting `GRUB_ENTRY_NAME` in the [cOS configuration file](../general_configuration) inside the derivative, or either via cloud-init before installation.
+The grub menu entry is generated during installation and can be configured by setting `GRUB_ENTRY_NAME` in the `/etc/os-release` file inside the derivative, or either via the [general configuration](../../customizing/general_configuration) to specify installation details.
 
-For example, specifying in `/etc/cos/config`:
+For example, specifying in `/etc/elemental/config.yaml`:
 
 ```bash
-GRUB_ENTRY_NAME=myOS
+install:
```

Comment:
maybe It's good to point out that applies also on upgrades somehow in the example, or @bk201 will have to ask some questions ;). This is currently used in harvester IIRC

Reply:
@mudler Thanks for pining. Yes, we use `GRUB_ENTRY_NAME` in `/etc/os-release` and need it to work between upgrades. From the revised document, this is still supported. Or did I get it wrong? Thanks!

Reply:
this is currently not supported in `elemental install` as is now in master, I am working on bringing that back.

Reply:
@bk201 the pr on rancher-sandbox/elemental#218 ports back this behavior, with a slightly difference I don't think it affects you in any way. 
The `GRUB_ENTRY_NAME` value that is honored during install is the one present within the installed system, not the `/etc/os-release` from the host (which is the same when installing from an installation live ISO).